### PR TITLE
fix: adds the POD_NAMESPACE to the RBD provisioner template

### PR DESCRIPTION
**Pull request by wilmardo**
Signed-off-by: wilmardo <info@wilmardenouden.nl>

As discussed on Slack, see these comment for more background information:
https://github.com/ceph/ceph-csi/issues/593#issuecomment-533598908

### CODE REVIEW

**Comment by wilmardo** [comment]
Is that really needed? #570 should be complete today, I added these changes there already.

Currently comparing my generated helm template with the current master and crosschecking between cephfs and rbd, therefore all those questions :)

**Comment by Madhu-1** [comment]
@wilmardo yes, lets track #570 for refactoring helm chats not to do any bug fix for kube templates

**Comment by Madhu-1** [comment]
@ShyamsundarR @humblec PTAL

**Comment by wilmardo** [comment]
@Madhu-1 Added to the helm charts as well.

**Comment by wilmardo** [comment]
Oh crap, thank for noticing! Will update this now :)

File: deploy/rbd/kubernetes/v1.13/csi-rbdplugin.yaml [hunk 1]
```diff
@@ -72,6 +72,10 @@ spec:
               valueFrom:
                 fieldRef:
                   fieldPath: spec.nodeName
+            - name: POD_NAMESPACE
```
Comment (Madhu-1) [inline deploy/rbd/kubernetes/v1.13/csi-rbdplugin.yaml]: not needed here,  we need pod namespace in provisioner, not in node plugin

**Comment by Madhu-1** [comment]
need to backport helm chat changes to release branch

**Pull request merged**